Cargo.toml

```diff
@@ -24,7 +24,7 @@ appveyor = { repository = "rust-random/rand" }
 # Meta-features:
 default = ["std", "std_rng"]
 nightly = ["simd_support"] # enables all features requiring nightly rust
-serde1 = [] # does nothing, deprecated
+serde1 = ["serde"]
```

Comment:
Correct me if I'm wrong, but this is a breaking change, isn't it? Builds previously working might fail now because of the new version requirement on serde.